cli/src/commands/branch.rs

```diff
@@ -77,6 +80,10 @@ pub struct BranchListArgs {
     /// wouldn't have a local target.
     #[arg(long, short)]
     revisions: Vec<RevisionArg>,
+
+    /// Whether to list non-tracking remote branches
```

Comment:
nit: I think we usually explain what it does when it's enabled (because there's no way to explicitly disable boolean flags with clap, I think), i.e. something like "List non-tracking remote branches". Or maybe "Also list ..." or "Include ..." to make it clearer that other branches are also included?